### ISSUE TITLE
void-variable cider--cljs-repl-types

### ISSUE THREAD

**Issue by manuel-uberti** [clojure-emacs/cider#2218]
## Expected behavior
Correctly start CIDER with both CLJ and CLJS REPLs.

## Actual behavior
```
Debugger entered--Lisp error: (void-variable cider--cljs-repl-types)
  cider-create-sibling-cljs-repl(#<buffer *cider-repl boodle*>)
  nrepl-server-filter(#<process nrepl-server> "nREPL server started on port 36626 on host :: - nrepl://:::36626\n")
```

## Steps to reproduce the problem
- visit a clj/cljs file in your project
- <kbd>M-x cider-jack-in-clojurescript</kdb>
- select `Figwhell-sidecar` at the CLJS REPL prompt

### CIDER version information

```
;; CIDER 0.17.0snapshot (package: 20180228.2012), nREPL 0.2.13
;; Clojure 1.9.0, Java 1.8.0_151
```

### Lein/Boot version

Leiningen 2.8.1 on Java 1.8.0_151 OpenJDK 64-Bit Server VM

### Emacs version

GNU Emacs 27.0.50 (build 1, x86_64-debian-linux-gnu, GTK+ Version 3.18.9) of 2018-02-28

### Operating system

Ubuntu 16.04

**Comment by manuel-uberti** [comment]
#2217 should be related.

**Comment by manuel-uberti** [comment]
Fixed by #2217.
